### DISCUSSION TITLE
Ability to add a header/footer

### DISCUSSION THREAD

**Discussion by IvanTahoe** [outline/outline#5327] [Ideas]
Ideally, this could be text or image / a combination of both. It would show once on the actual wiki pages, and then when printing, it would be repeated across all the pages. We're using this for internal documentation, being able to have our logo on every page that prints would be awesome!!

**Comment by infogulch** [comment]
Customizing the print template for certain pages would be make Outline a compelling tool for documentation that needs to be maintained in print form.

**Comment by awalser1** [comment]
Hi Team
As already mentioned I would also see a huge benefit having customisable header and footer on document printing and exporting. We use outline as documentation tool for our feasibility studies. Due to the fact that most of the customers want to have a final report as pdf, a header with our logo and text, also including some document fields like page number, total of pages, would be highly appreciated.
Are there any plans to integrate this functionality in the future? Thank you in advance for your feedback.
Kindly regards,
Andreas

**Comment by infogulch** [comment]
I opened a new discussion about one way this could be implemented. I go pretty deep into demonstrating its viability and include a demo of what it looks like:

https://github.com/outline/outline/discussions/10156
